tests/modules/list.c

```diff
@@ -120,7 +120,18 @@ int list_edit(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
         }
     }
 
+    RedisModuleString *v = RedisModule_ListGet(key, index);
+    RedisModule_ReplyWithArray(ctx, v ? 6 : 4);
+    RedisModule_ReplyWithCString(ctx, "num_edits");
     RedisModule_ReplyWithLongLong(ctx, num_edits);
+    RedisModule_ReplyWithCString(ctx, "index");
+    RedisModule_ReplyWithLongLong(ctx, index);
+    if (v) {
+        RedisModule_ReplyWithCString(ctx, "entry");
+        RedisModule_ReplyWithString(ctx, v);
+        RedisModule_FreeString(ctx, v);
+    } 
+
```

Comment:
Please also update the comment before `list_edit()` to match this. Currently it says "The reply is the number of edits (inserts + replaces + deletes) performed.".

Reply:
If we want to return a map, how about also returning a separate counter for "num_inserts", "num_deletes" and "num_replaces" instead of the sum "num_edits"?

Reply:
Done.